### PR TITLE
LG-2871 LG-2662 updates toast design

### PR DESCRIPTION
**Pull request by TheSonOfThomp**
## ✍️ Proposed changes

Updates basic styling of Toast. Demo: https://thesonofthomp.github.io/leafygreen-ui/?path=/story/components-toast--dismissible

Note: This does _not_ incorporate any new multi-toast behavior

🎟 _Jira ticket:_ 
- [Update Toast design](https://jira.mongodb.org/browse/LG-2871)
- [Reorg. internal directory](https://jira.mongodb.org/browse/LG-2662)

Screen Shot 2023-02-17 at 6 32 03 PM

## 🛠 Types of changes

- [x] Tooling (updates to workspace config or internal tooling)
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Style change (purely visual updates; if this is the case, attach a screenshot below!)

## ✅ Checklist

- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [x] I have run `yarn changeset` and documented my changes

- [ ] I have added my new package to the global tsconfig
- [ ] I have added my new package to the Table of Contents on the global README

## Feedback Levels

Please try to mark your comments with one of these levels

1. Praise (👏 🚀 🔥 🍾)
2. Opinions, Ideas, Suggestions (💡 👀)
3. Let’s discuss @ Code review (🤷 🤔 🧐 )
4. Doesn’t meet agreed upon standards (😬 ⚠️ ‼️)
5. Missing an edge case (🛑 🙅 ❌)
6. Missing core functionality (⛔️ ☣️ 🚧 🌋 💀)

(Note: Use ⌘-⌃-␣ to open the Emoji keyboard!)

### CODE REVIEW

**Review by sandynguyenn — APPROVED** [review]
Looks great! Praise (👏 🚀 🔥 🍾)

File: .changeset/big-parrots-scream.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+'@leafygreen-ui/toast': major
+---
+
+Updates Toast design to align with Toast v3 in Figma. Minor API updates to prepare for multi-toast behavior.
+- Makes `title` prop required (instead of `body`)
+- Renames `body` to `description` to align with Figma component
```
Comment (brookescarlett) [inline .changeset/big-parrots-scream.md:7]: Designers will make the body <=> description change on their side

Comment (brookescarlett) [reply .changeset/big-parrots-scream.md:7]: So let's leave this as is

File: packages/toast/src/ProgressBar/ProgressBar.tsx [hunk 2]
```diff
@@ -59,7 +57,7 @@ const progressBarThemeStyle: Record<Theme, string> = {
       #083c90 ${toastWidth}px
     );
   `,
-  [Theme.Light]: css`
+  [Theme.Dark]: css`
     background-color: #0498ec;
     background-image: linear-gradient(
       90deg,
```
Comment (spark33) [inline packages/toast/src/ProgressBar/ProgressBar.tsx:63]: Let's move this to a `ProgressBar.styles.ts` if we want to call the restructuring done

File: packages/toast/src/Toast/Toast.tsx [hunk 3]
```diff
@@ -58,65 +67,55 @@ function Toast({
             <div
               ref={nodeRef}
               className={cx(
-                baseElementStyles.toast,
+                baseToastStyle,
```
Comment (spark33) [inline packages/toast/src/Toast/Toast.tsx:70]: ignorable nit: `Styles`, for all style variables

**Review by spark33 — CHANGES_REQUESTED** [review]

**Review by shaneeza — COMMENTED** [review]
Looking good. One thing I noticed is that the focus and hover styles for links are not showing up.

**Review by spark33 — APPROVED** [review]

**Pull request merged**